experiments/ClimaEarth/run_amip.jl

```diff
@@ -378,14 +378,14 @@ elseif mode_name in ("slabplanet", "slabplanet_aqua", "slabplanet_terra")
 
     ## sea ice stub (here set to zero area coverage)
     ice_sim = Interfacer.SurfaceStub((;
-        T_sfc = CC.Fields.ones(boundary_space),
-        ρ_sfc = CC.Fields.zeros(boundary_space),
+        T_sfc = ones(boundary_space),
+        ρ_sfc = zeros(boundary_space),
         z0m = FT(0),
         z0b = FT(0),
         beta = FT(1),
-        α_direct = CC.Fields.ones(boundary_space) .* FT(1),
-        α_diffuse = CC.Fields.ones(boundary_space) .* FT(1),
-        area_fraction = CC.Fields.zeros(boundary_space),
+        α_direct = ones(boundary_space) .* FT(1),
```

Comment:
```suggestion
        α_direct = ones(boundary_space),
```

Reply:
I don't think we need this? `ones` in ClimaCore already inherits the undertype of the space:
```julia
function Base.ones(::Type{FT}, space::AbstractSpace) where {FT}
    field = Field(FT, space)
    data = parent(field)
    fill!(data, one(eltype(data)))
    return field
end
Base.ones(space::AbstractSpace) = ones(Spaces.undertype(space), space)
```